hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMRatisServerImpl.java

```diff
@@ -110,4 +111,18 @@ public void stop() throws IOException {
     server.close();
   }
 
+  @Override
+  public RaftServer getServer() {
+    return server;
+  }
+
+  @Override
+  public RaftGroupId getRaftGroupId() {
+    return raftGroupId;
+  }
+
+  @Override
+  public List<RaftPeer> getRaftPeers() {
+    return Collections.singletonList(new RaftPeer(raftPeerId));
```

Comment:
why ScmRatiSserver only have one entry in the raft peers, is this something we plan to implement in future?

Reply:
Yes. And this class will be deprecated soon in SCM group feature.